clang/test/CheckedCRewriter/checkedregionvariadic.c

```diff
@@ -0,0 +1,10 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+#include <stdio_checked.h>
+
+void foo(int *i) { 
+  //CHECK: void foo(_Ptr<int> i) _Checked {
+  *i = 3;
+  printf("Test");
+  //CHECK: _Unchecked { printf("Test"); };
```

Comment:
Neat! Does this work generally, or just for variadic functions specially?

Reply:
Just for variadic, although we could explore more general ideas. I noticed this because there were a lot of functions that were completely safe except for a call to printf